src/__tests__/lib/containers/personal_access_token/AccessTokenPage.test.tsx

```diff
@@ -1,17 +1,49 @@
-import { act } from '@testing-library/react'
```

Comment:
Fixed unstable tests by using mocks of components that have their own test suite and using RTL instead of enzyme. Test cases are the same.